en/gps_compass/septentrio.md

```diff
@@ -28,19 +28,23 @@ Certain receivers are recommended for autopilot applications because of their ph
 - RTK positioning using corrections over MAVLink from a ground control station
 - Dual-antenna heading with mosaic-H-based receivers
 - Moving base heading with two single-antenna receivers
-- SBF logging to internal storage on the receiver
+- Flexible SBF logging to internal storage on the receiver
```

Comment:
That's cool. As a reader I'm wondering why I care? Is there a link to information on how you might use these logs we could provide?